pkg/operator/ceph/cluster/osd/deviceSet.go

```diff
@@ -200,7 +201,9 @@ func (c *Cluster) createDeviceSetPVC(existingPVCs map[string]*v1.PersistentVolum
 		pvcID = deviceSetPVCID(deviceSetName, pvcTemplate.GetName(), setIndex)
 		existingPVC = existingPVCs[pvcID]
 	}
-	pvc := makeDeviceSetPVC(deviceSetName, pvcID, setIndex, pvcTemplate, c.clusterInfo.Namespace)
+	re := regexp.MustCompile("[/:]")
```

Comment:
If the ceph image version is invalid as a label value and has unacceptable characters, Rook panics here. How about leaving an error log and continuing to deploy OSD without applying the label?